guides/common/modules/proc_creating-a-job-template.adoc

```diff
@@ -23,6 +23,8 @@ For examples, see the *Help* tab.
 . Optional: Click *Foreign input set* to include other templates in this job.
 . Optional: In the *Effective user* area, configure a user if the command cannot use the default `remote_execution_effective_user` setting.
 . Optional: If this template is a snippet to be included in other templates, click the *Type* tab and select *Snippet*.
+. Optional: If you use the Ansible provider, click the *Ansible* tab.
+Select *Enable Ansible Callback* to allow hosts to send facts and reports back to {Project} after a job finishes.
```

Comment:
```suggestion
Select *Enable Ansible Callback* to allow hosts to send facts and report back to {Project} after a job finishes.
```

Reply:
Thanks, but I will stick with "reports", because I'm not trying to say that "hosts report back", but "hosts send back reports".

Reply:
Actually, I will rephrase it altogether, because I have new intel coming in.

Reply:
Ack!